### PR TITLE
Add library dependencies

### PR DESCRIPTION
**Pull request by tvealot**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
         
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.2'
+        classpath "com.android.tools.build:gradle:$build_tools_version"
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath "org.jetbrains.kotlin:kotlin-serialization:$kotlin_version"
+
+        classpath "org.jetbrains.kotlin:kotlin-noarg:$kotlin_version"
+        classpath "androidx.navigation:navigation-safe-args-gradle-plugin:$navigation_safe_args_version"
```
Comment (NoSurprises) [inline build.gradle]: Do we really need it?

File: app/build.gradle [hunk 2]
```diff
@@ -1,34 +1,13 @@
...
-        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
-    }
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
-        }
-    }
-}
+apply plugin: "androidx.navigation.safeargs.kotlin"
```
Comment (NoSurprises) [inline app/build.gradle]: And this

File: build.gradle (hunk 1 above)
Comment (tvealot) [reply build.gradle]: done

File: app/build.gradle (hunk 2 above)
Comment (tvealot) [reply app/build.gradle]: done

**Pull request merged**